<launch>
  <node name="find_object_3d" pkg="find_object_2d" type="find_object_2d" output="screen">
    <param name="gui" value="true" type="bool"/>
    <param name="settings_path" value="$(find rtabmap)/launch/config/azimut3_find_object.ini" type="str"/>
    <param name="subscribe_depth" value="true" type="bool"/>
    <param name="objects_path" value="$(find rtabmap)/launch/config/books" type="str"/>
		
    <remap from="rgb/image_rect_color" to="camera/data_throttled_image"/>
    <remap from="depth_registered/image_raw" to="camera/data_throttled_image_depth"/>
    <remap from="depth_registered/camera_info" to="camera/data_throttled_camera_info"/>

    <param name="rgb/image_transport" type="string" value="compressed"/>
    <param name="depth/image_transport" type="string" value="compressedDepth"/>
  </node>
</launch>
